docs: review documentation files (#787)
MobilityData/gtfs-validator: docs/CONTRIBUTING.md

# Contribution guidelines 

## How to contribute to project? 

> All external contribution to this project is welcome. To propose changes we encourage contributors to open a new branch and propose their changes by opening a new PR. 

Reviewer: To proposes changes, we encourage

> All external contribution to this project is welcome. To propose changes, we encourage contributors to open a new branch and propose their changes by opening a new PR. 

Reviewer 2: ```suggestion
All contributions to this project are welcome. To propose changes, we encourage contributors to:
1. [Fork](https://docs.github.com/en/github/getting-started-with-github/fork-a-repo) this project on GitHub
1. Create a new branch, and
1. Propose their changes by opening a [new pull request](https://docs.github.com/en/github/collaborating-with-issues-and-pull-requests/about-pull-requests). 

If you're looking for somewhere to start, check out the issues labeled ["Good first issue"](https://github.com/MobilityData/gtfs-validator/issues?q=is%3Aopen+is%3Aissue+label%3A%22good+first+issue%22) or [Community](https://github.com/MobilityData/gtfs-validator/issues?q=is%3Aopen+is%3Aissue+label%3Acommunity).
```

## Having problems?
Have you encountered an error? A critical step in troubleshooting is being able to reproduce the problem. Instructions to publicly reproduce errors using GitHub Actions can be found in our [guide to reproduce errors.](/docs/REPRODUCE_ERRORS.md)

## Code template

> ###Issue and PR templates

Reviewer 2: ```suggestion
### Issue and PR templates
```

Reviewer 2: I would remove "issue and PR templates" from the code template section, as that doesn't really fit, and instead just move the text under the "How to contribute to the project?" section right after the mention of the pull request. And given that GitHub forces you to do most of this, I think this text can be reduced to:

>We encourage contributors to format pull request titles following the [Conventional Commit Specification](https://www.conventionalcommits.org/en/v1.0.0/)

> Issue templates have been designed to ease the processes to suggest a new feature or report a bug. We encourage contributors to not only format their PR's title following the [Conventional Commit Specification](https://www.conventionalcommits.org/en/v1.0.0/), but also use the PR templates made available on this repository.

Reviewer: We encourage contributors not only to format ..., but also to use

> ###Coding style

Reviewer 2: ```suggestion
### Coding style
```

> Sticking to a single consistent and documented coding style for this project is important to ensure that code reviewers dedicate their attention to the functionality of the validation, as opposed to disagreements about the coding style (and avoid bike-shedding). 

Reviewer 2: I assume https://en.wikipedia.org/wiki/Law_of_triviality is "bike-shedding" here. Please link to it.

Google Java Style has been chosen for this project, therefore we suggest future contributors to maintain the same style for the consistency of the codebase. 

> Developers should refer to Google Java Style Guide (https://google.github.io/styleguide/javaguide.html) for more information. Automated code formatting plugins for popular Java IDEs is available here: https://github.com/google/google-java-format.

Reviewer: plugins for popular Java IDEs are available

> Google Java Style has been chosen for this project, therefore we suggest future contributors to maintain the same style for the consistency of the codebase. 
> Developers should refer to Google Java Style Guide (https://google.github.io/styleguide/javaguide.html) for more information. Automated code formatting plugins for popular Java IDEs are available here: https://github.com/google/google-java-format.

Reviewer 2: ```suggestion
This project uses the [Google Java Style](https://google.github.io/styleguide/javaguide.html). IDE plugins to automatically format your code in this style are [here](https://github.com/google/google-java-format).
```

> ## How to add a new validation rule?

Reviewer 2: ```suggestion
## How do I add a new validation rule?
```

> [NEW_RULES.md](/docs/NEW_RULES.md) gathers instructions to add new validation rules to the validator. Please refer to this documentation for more information.

Reviewer 2: ```suggestion
The ["Adding new rules" documentation](/docs/NEW_RULES.md) includes instructions for adding new validation rules to the validator.
```
